src/transformers/models/gemma/modeling_gemma.py

```diff
@@ -13,7 +13,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-""" PyTorch Gemma model."""
+"""PyTorch Gemma model."""
```

Comment:
unrelated changes should not be part of the PR

Reply:
Oops, I think Ruff changed that and I'll remove it

Reply:
ah I should have used

    make fixup

not ruff, I think